src/wallet/wallet2.h

```diff
@@ -1623,6 +1625,9 @@ namespace tools
     std::unique_ptr<wallet_device_callback> m_device_callback;
 
     ExportFormat m_export_format;
+
+    static boost::mutex default_daemon_address_lock;
```

Comment:
I'll leave it there since it's used in the header. Guess I could move that to the cpp, but the string would still be in the class (guess it could be made global too but meh).